wolfssl/wolfcrypt/asn.h

```diff
@@ -639,6 +639,9 @@ WOLFSSL_LOCAL int ExtractDate(const unsigned char* date, unsigned char format,
 WOLFSSL_LOCAL int ValidateDate(const byte* date, byte format, int dateType);
 
 /* ASN.1 helper functions */
+#ifdef WOLFSSL_CERT_GEN
+WOLFSSL_TEST_API int SetName(byte* output, word32 outputSz, CertName* name);
```

Comment:
See ZenDesk #2137. This seems to be the only way to convert a subject name to DER. I could instead add a new function to wrap it. This is only exposed with the "WOLFSSL_TEST_CERT" which this customer is using for their OpenSSL compatibility additions.